### PR TITLE
UAF-1347 UAF-1686 UAF-1687 Unschedule all scheduled jobs

### PR DESCRIPTION
**Pull request by adamkost**

### CODE REVIEW

**Comment by adamkost** [comment]
Per Heather, hold off on the merge until after release 4 is finished.

**Comment by ghost** [comment]
Please move your overrides to edu/arizona as this is a UA specific mod, not a bug. 
Please do not delete the beans defining the triggers, just remove them from each Spring file's trigger names list.

**Comment by heather-lo** [comment]
To support what Julie asserted about how we should keep the beans that define the triggers, but remove them from the list in the triggerNames property, see the documentation here: https://wiki.kuali.org/pages/viewpage.action?title=Batch+6&spaceKey=KULDOC#Batch6-Definingatrigger

**Comment by adamkost** [comment]
Created override beans for all *Job beans by copying the original and changing the parent from parent="scheduledJobDescriptor" to parent="unscheduledJobDescriptor". Most of theses were not abstract beans, so had to be copied in their entirety.
Also overrode the triggerNames property of the configuration bean to have an empty list.

File: kfs-ar/src/main/resources/edu/arizona/kfs/module/ar/spring-ar.xml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    <bean id="upcomingMilestoneNotificationJob"                 parent="unscheduledJobDescriptor">
+        <property name="steps">
+            <list>
+                <ref bean="upcomingMilestoneNotificationStep" />
+            </list>
+        </property>
+    </bean>
+
+    <bean id="clearContractsGrantsInvoiceDocumentErrorLogJob"   parent="unscheduledJobDescriptor">
```
Comment (ghost) [inline kfs-ar/src/main/resources/edu/arizona/kfs/module/ar/spring-ar.xml]: This job was already unscheduled, so no override needed. Also, please remove the extra spaces or tabs between bean id and parent on the other jobs (to improve readability).

File: kfs-core/src/main/resources/edu/arizona/kfs/coa/spring-coa.xml [hunk 2]
```diff
@@ -21,6 +21,10 @@
                 <value>edu/arizona/kfs/coa/ojb-coa.xml</value>
             </list>
         </property>
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/coa/spring-coa.xml]: There aren't actually any triggers defined in spring-coa, so this is an unneeded override.

File: kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml [hunk 3]
```diff
@@ -28,6 +28,10 @@
                 <value>edu/arizona/kfs/fp/ojb-fp.xml</value> 
             </list>
         </property>
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml]: There aren't actually any triggers defined in spring-fp, so this is an unneeded override.

File: kfs-core/src/main/resources/edu/arizona/kfs/gl/spring-gl.xml [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+                           http://www.springframework.org/schema/tx
+                           http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
+                           http://www.springframework.org/schema/aop
+                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+    <bean id="glModuleConfiguration" parent="glModuleConfiguration-parentBean" >
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/gl/spring-gl.xml]: There aren't actually any triggers defined in spring-gl, so this is an unneeded override.

File: kfs-ec/src/main/resources/edu/arizona/kfs/module/ec/spring-ec.xml [hunk 5]
```diff
@@ -16,5 +16,9 @@
         		<value>classpath:edu/arizona/kfs/module/ec/document/datadictionary/*.xml</value>
         	</list>
         </property>
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-ec/src/main/resources/edu/arizona/kfs/module/ec/spring-ec.xml]: There aren't actually any triggers defined in spring-ec, so this is an unneeded override.

File: kfs-ld/src/main/resources/edu/arizona/kfs/module/ld/spring-ld.xml [hunk 6]
```diff
@@ -21,5 +21,103 @@
                 <value>classpath:org/kuali/kfs/module/ld/document/datadictionary/overrides/*.xml</value>
             </list>
         </property>
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-ld/src/main/resources/edu/arizona/kfs/module/ld/spring-ld.xml]: There aren't actually any triggers defined in spring-ld, so this is an unneeded override.

File: kfs-tem/src/main/resources/edu/arizona/kfs/module/tem/spring-tem.xml [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+                           http://www.springframework.org/schema/tx
+                           http://www.springframework.org/schema/tx/spring-tx-2.0.xsd
+                           http://www.springframework.org/schema/aop
+                           http://www.springframework.org/schema/aop/spring-aop-2.0.xsd">
+
+    <bean id="temModuleConfiguration" parent="temModuleConfiguration-parentBean"  >
+        <property name="triggerNames">
```
Comment (ghost) [inline kfs-tem/src/main/resources/edu/arizona/kfs/module/tem/spring-tem.xml]: There aren't actually any triggers defined in spring-tem, so this is an unneeded override.

**Comment by ghost** [comment]
I have completed my initial review. Please see my comments about unneeded trigger overrides.
Thanks.

File: kfs-core/src/main/resources/institutional-config.properties [hunk 8]
```diff
@@ -31,4 +39,4 @@ spring.source.files=classpath:spring-kfs-imported-rice-beans.xml, \
   classpath:edu/arizona/kfs/rq/spring-rq.xml, \
   classpath:edu/arizona/kfs/module/purap/spring-purap.xml, \
   classpath:edu/arizona/kfs/sys/spring-sys.xml, \
-  classpath:edu/arizona/kfs/module/ec/spring-ec.xml
\ No newline at end of file
+  classpath:edu/arizona/kfs/module/ec/spring-ec.xml
```
Comment (ghost) [inline kfs-core/src/main/resources/institutional-config.properties]: This is a duplicate (see line 29), please remove.

**Comment by ghost** [comment]
I have completed my second review. Please see my comment.

**Comment by ghost** [comment]
Looks good!

**Pull request merged**